includes/admin/views/omise-page-settings.php

```diff
@@ -98,85 +98,91 @@
-					<td>
-						<table class="widefat fixed striped" cellspacing="0">
-							<thead>
-								<tr>
-									<?php
-										$columns = array(
-											'name'    => __( 'Payment Method', 'omise' ),
-											'status'  => __( 'Enabled', 'omise' ),
-											'setting' => ''
-										);
+		<?php if ($settings['account_country']) : ?>
+			<p><?php _e( 'The table below is a list of available payment methods that you can enable in your WooCommerce store.', 'omise' ); ?></p>
+			<table class="form-table">
+				<tbody>
+					<tr>
+						<th scope="row"><label for="sandbox"><?php _e( 'Available Payment Methods', 'omise' ); ?></label></th>
+						<td>
+							<table class="widefat fixed striped" cellspacing="0">
+								<thead>
+									<tr>
+										<?php
+											$columns = array(
+												'name'    => __( 'Payment Method', 'omise' ),
+												'status'  => __( 'Enabled', 'omise' ),
+												'setting' => ''
+											);
 
-										foreach ( $columns as $key => $column ) {
-											switch ( $key ) {
-												case 'status' :
-												case 'setting' :
-													echo '<th style="text-align: center; padding: 10px;" class="' . esc_attr( $key ) . '">' . esc_html( $column ) . '</th>';
-													break;
+											foreach ( $columns as $key => $column ) {
+												switch ( $key ) {
+													case 'status' :
+													case 'setting' :
+														echo '<th style="text-align: center; padding: 10px;" class="' . esc_attr( $key ) . '">' . esc_html( $column ) . '</th>';
+														break;
 
-												default:
-													echo '<th style="padding: 10px;" class="' . esc_attr( $key ) . '">' . esc_html( $column ) . '</th>';
-													break;
-											}
+													default:
+														echo '<th style="padding: 10px;" class="' . esc_attr( $key ) . '">' . esc_html( $column ) . '</th>';
+														break;
+												}
 
-										}
-									?>
-								</tr>
-							</thead>
-							<tbody>
-								<?php
-								$available_gateways = array(
-									new Omise_Payment_Alipay,
-									new Omise_Payment_Billpayment_Tesco,
-									new Omise_Payment_Creditcard,
-									new Omise_Payment_Installment,
-									new Omise_Payment_Internetbanking,
-									new Omise_Payment_Truemoney
-								);
-								foreach ( $available_gateways as $gateway ) :
+											}
+										?>
+									</tr>
+								</thead>
+								<tbody>
+									<?php
+									$available_gateways = array(
+										new Omise_Payment_Alipay,
+										new Omise_Payment_Billpayment_Tesco,
+										new Omise_Payment_Creditcard,
+										new Omise_Payment_Installment,
+										new Omise_Payment_Internetbanking,
+										new Omise_Payment_Truemoney
+									);
+									foreach ( $available_gateways as $gateway ) :
+										if ( $gateway->is_country_support( $settings['account_country'] ) ) :
```

Comment:
Note; continue from https://github.com/omise/omise-woocommerce/pull/151/files#r368338668
Here is another line that has been added. The rest is code indentation.